woodwork/statistics_utils.py

```diff
@@ -106,6 +124,20 @@ def _get_describe_dict(dataframe, include=None, callback=None):
         values["physical_type"] = series.dtype
         values["logical_type"] = logical_type
         values["semantic_tags"] = semantic_tags
+
+        # Calculate extra detailed stats, if requested
+        if extra_stats:
+            if column.is_numeric:
+                values["histogram"] = _get_histogram_values(series, bins=bins)
+                if isinstance(column.logical_type, (Integer, IntegerNullable)):
+                    _range = range(int(values["min"]), int(values["max"]) + 1)
+                    if len(_range) <= bins:
+                        values["top_values"] = _get_top_values_numeric(series, _range)
+            elif column.is_categorical:
+                values["top_values"] = _get_top_values_categorical(series, top_x)
```

Comment:
Likely not worth the extra logic required of this, because it seems we don't include the index column, but if the column is unique, we probably don't need to calculate top values and maybe histogram. Since we've already calculated count and nunique it wouldn't take additional computation to check.